keyboards/playkbtw/ca66/keymaps/via/keymap.c

```diff
@@ -0,0 +1,45 @@
+/*
+*Based on the KeLorean CA66 keymap.
+*Made for HHKB layout with CA66 badge in lower left corner and split spacebars.
+*Left Spacebar - tap for backspace, hold for Layer1
+*Right Spacebar - tap for space, hold for Layer2
+*Layer 2 is same as Layer 1 except Left Spacebar is now delete
+*Cap Locks has been replaced with Momentary Toggle to Layer3
+*Layer 3 tab is the soft reset.
+*/
```

Comment:
```suggestion
  
/*
Copyright 2021 J.Flanagan
This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 2 of the License, or
(at your option) any later version.
This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.
You should have received a copy of the GNU General Public License
along with this program.  If not, see <http://www.gnu.org/licenses/>.
*/
```